src/cli/index.js

```diff
@@ -24,30 +24,34 @@ export default (config, http) => {
 
   const generateStats = async (email, year, month) => {
     logger.info(`Generating stats for ${year}-${month}`);
-    await app.generateStats(year, month, email);
+    await application(config, http, slack).generateStats(year, month, email);
     logger.info(`Sent stats report to ${email}`);
   };
 
-  const generateBillingReports = async (email, year, month, lastNames) => {
+  const generateBillingReports = async (email, year, month, lastNames, opts) => {
     logger.info(`Generating billing reports for ${year}-${month}`);
-    await app.generateBillingReports(year, month, lastNames, email);
+
+    const { harvest } = opts;
+    const harvestAccount = harvest && harvest.match(/^(witted|mavericks)$/g) ? harvest : 'mavericks';
```

Comment:
Maybe this default selection could be moved to own function, as it seems to be used twice in PR. Would also help if this is used by someone else.

Reply:
Related to comment above, second check was unnecessary and removed since setting have no need to know or make decisions based on selected account.